Added a task to check if job id exists (#39796)
demisto/content: Packs/PAN-OS/ReleaseNotes/2_3_12.md

#### Playbooks

##### PAN-OS Commit Configuration v2

> - Updated the PAN-OS Commit Configuration v2 playbook to check if job id exists before the next task.

Reviewer: ```suggestion
- Updated the PAN-OS Commit Configuration v2 playbook to check for any changes to commit by checking the job ID.
```
